packages/admin/src/pages/dashboard/index.page.tsx

```diff
@@ -93,4 +93,22 @@ const Dashboard = ({
   );
 };
 
+const getBannerProps = ({
+  findMigrationStatus,
+  applyMigrationStatus,
+}: Record<string, string>) => {
+  if (process.env.ONE_LOGIN_MIGRATION_JOURNEY_ENABLED !== 'true') return null;
+  if (findMigrationStatus === FAILED || applyMigrationStatus === FAILED)
+    return FAILED;
+
+  if (findMigrationStatus === SUCCEEDED || applyMigrationStatus === SUCCEEDED)
```

Comment:
code golf but you can do

```suggestion
  if ([findMigrationStatus, applyMigrationStatus].includes(FAILED))
    return FAILED;

  if ([findMigrationStatus, applyMigrationStatus].includes(SUCCEEDED))
```